src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -434,6 +434,65 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    validator=get_permission_validator(t_account_permissions))
         c.ignore('sas_token')
 
+    or_policy_type = CLIArgumentType(
+        options_list=['--policy', '-p'],
+        help='The object replication policy definition between two storage accounts, in JSON format. '
+             'Multiple rules can be defined in one policy.'
+    )
+    policy_id_type = CLIArgumentType(
+        options_list=['--policy-id'],
+        help='The ID of object replication policy or "default" if the policy ID is unknown.'
+    )
+    rule_id_type = CLIArgumentType(
+        options_list=['--rule-id', '-r'],
+        help='Rule Id is auto-generated for each new rule on destination account. It is required '
+             'for put policy on source account.'
+    )
+    prefix_math_type = CLIArgumentType(
+        nargs='+', arg_group='Filters', options_list=['--prefix-match', '--prefix'],
+        help='Optional. Filter the results to replicate only blobs whose names begin with the specified '
+             'prefix.'
+    )
+    min_creation_time_type = CLIArgumentType(
+        options_list=['--min-creation-time', '-t'], arg_group='Filters', type=get_datetime_type(True),
+        help="Blobs created after the time will be replicated to the destination. It must be in datetime format "
+             "'yyyy-MM-ddTHH:mm:ssZ'. Example: 2020-02-19T16:05:00Z")
+
+    with self.argument_context('storage account or-policy') as c:
+        c.argument('account_name', acct_name_type, id_part=None)
+        c.argument('resource_group_name', required=False, validator=process_resource_group)
+        c.argument('object_replication_policy_id', policy_id_type)
+        c.argument('policy_id', policy_id_type)
+        c.argument('source_account', options_list=['--source-account', '-s'],
+                   help='The source storage account name. Required when no --policy provided.')
+        c.argument('destination_account', options_list=['--destination-account', '-d'],
+                   help='The destination storage account name. Apply --account-name value as destination account '
+                   'when there is no destination account provided in --policy and --destination-account.')
+        c.argument('properties', or_policy_type)
```

Comment:
Is it better to define it as "policy" which is more clear instead of "properties"?

Reply:
properties is in SDK method but CLI parameter is `--policy`